docs/ci.md

```diff
@@ -117,3 +117,13 @@ begin and end so it's easier for those involved in the PR to follow the results.
 Here is an example of using the GitHub Web UI to launch an E2E workflow:
 
 ![GitHub Actions Run Workflow Example](images/github-actions-run-workflow-example.png)
+
+## Dependency on external Pull Requests
```

Comment:
This is really good, but can you also update the contributing guide to briefly mention the `depends-on-action` and why/how you might want to use it? -> https://github.com/instructlab/instructlab/blob/main/CONTRIBUTING/CONTRIBUTING.md

It doesn't have to be super detailed.

Reply:
done